deeplake/auto/unstructured/coco/convert.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+from typing import Dict, Any, Optional
+
+from deeplake.core.tensor import Tensor
+
+
+def coco_to_deeplake(
+    coco_key: str,
+    value: Any,
+    destination_tensor: Tensor,
+    category_lookup: Optional[Dict] = None,
+):
+    """Takes a key-value pair from coco data and converts it to data in Deep Lake compatible format"""
+    dtype = destination_tensor.meta.dtype
+
+    if coco_key == "bbox":
+        assert len(value) == 4
+        return np.array(value, dtype=dtype)
+    elif coco_key == "segmentation":
+        try:
+            # Currently having only ONE polygon is supported.
+            # Multiple polygons are under same label, but there is only a single bbox.
```

Comment:
has this been discussed with Ivo?

Reply:
Yep. For now added a warning indicating that the data is skipped